### PR TITLE
NFDIV-3090 - Prevent AOS being updated after submit

### PR DESCRIPTION
**Pull request by danielbe11**
### Change description ###

- It is possible to go back to AOS URLs on the FE to change answers. This shouldn't be possible.

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-3090

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/main/steps/index.utils.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { CaseWithId } from '../app/case/case';
+
+export const hasSubmittedAos = (userCase: CaseWithId): boolean => {
```
Comment (benouaer) [inline src/main/steps/index.utils.ts:3]: do we need a whole new utils file for just a boolean? it's only used in one place for now right? so we could put this function in the index file

File: src/main/steps/respondentSequence.ts [hunk 2]
```diff
@@ -89,7 +89,6 @@ const sequence: Step[] = [
 
 export const respondentSequence = ((): Step[] => {
   return sequence.map(step => ({
-    ...step,
```
Comment (benouaer) [inline src/main/steps/respondentSequence.ts:92]: do we not need the other step properties?

**Review by benouaer — APPROVED** [review]

File: src/main/steps/index.utils.ts (hunk 1 above)
Comment (danielbe11) [reply src/main/steps/index.utils.ts:3]: Moved it to index

File: src/main/steps/respondentSequence.ts (hunk 2 above)
Comment (danielbe11) [reply src/main/steps/respondentSequence.ts:92]: This ...step was added when our steps looked like this:
  {
    url: DETAILS_OTHER_PROCEEDINGS,
    showInSection: Sections.OtherCourtCases,
    getNextStep: () => HOW_THE_COURTS_WILL_CONTACT_YOU,
  },
but now we don't have showInSection we can remove it.

**Pull request merged**